docs/ss_slide_004.md

```diff
@@ -2,10 +2,10 @@
 
 
 {:style="text-align:center;"}
-![lenia title](https://raw.githubusercontent.com/riveSunder/yuca/master/assets/consequential_step_size/adaptive_ss_win.gif)
+![lenia title](https://raw.githubusercontent.com/riveSunder/yuca_docs/master/assets/consequential_step_size/adaptive_ss_win.gif)
 This pattern, here running with step size 0.1 in a CA called s613, starts to wobble after ~100 steps or so, and eventually becomes unstable. 
 {:style="text-align:center;"}
 
 
 {:style="text-align:center;"}
-[Previous slide](https://rivesunder.github.io/yuca/ss_slide_003.md) -- [Next slide](https://rivesunder.github.io/yuca/ss_slide_004b)
+[Previous slide](https://rivesunder.github.io/yuca_docs/ss_slide_003.md) -- [Next slide](https://rivesunder.github.io/yuca_docs/ss_slide_004b)
```

Comment:
Let's go ahead and revert the links to the gh-pages to point to `https://rivesunder.github.io/yuca` instead of yuca_docs. This should decrease navigation confusion while still offloading the larger assets storage to yuca_docs.